### ISSUE TITLE
Fullscreen: Tab Based Settings for Scrolling Direction

### ISSUE THREAD

**Issue by CriticalComposer** [JosefNemec/Playnite#1619]
I'd like to be able to set the Scrolling Direction per tab in Fullscreen mode. Both the PS4 and the Switch use horizontal scrolling for their main view and switch to vertical scrolling when navigating your entire library. I'd like to replicate this in Playnite without having to manually switch it all of the time.

**Comment by JosefNemec** [comment]
Personally I don't think this makes sense with current UX. Fullscreen tabs are basically just separate filters and I'm not sure it's worth having different view settings based on applied filter.

**Comment by CriticalComposer** [comment]
I agree that this is a strange request but I figured it would be easier than what I actually desire regarding the UX. Right now I use a lower number of columns than rows which allows me to switch between two "modes": one for low volume filters and one for high volume filters. Ideally I'd like to automate the number of rows or columns based on the number of items in an applied filter. Having a single threshold -- such as less than 20 and 20 or more -- would be enough but being able to set more points would be better. 

Basically, I don't like seeing empty space on low volume filters but I also want to maximize the amount of titles on screen when I'm navigating my entire library. Even just having an algorithm to adjust the columns/rows for low volume filters would be amazing. If any of that didn't make sense just let me know and I can make a quick video about it -- showing off my current settings and providing a visual to my issues.
